tests/test_web_server.py

```
@@ -53,19 +233,20 @@ def test_validate_startup_security_allows_local_without_auth(monkeypatch):
 
 
 def test_validate_startup_security_blocks_non_local_without_auth(monkeypatch):
-    monkeypatch.delenv("PENGUIN_AUTH_ENABLED", raising=False)
+    monkeypatch.setenv("PENGUIN_AUTH_ENABLED", "false")
     monkeypatch.delenv("PENGUIN_ALLOW_INSECURE_NO_AUTH", raising=False)
 
     try:
         server.validate_startup_security("0.0.0.0")
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)